test/test_tree/Makefile

```diff
@@ -1,4 +1,6 @@
-test: test1 test2 test3 test4 test5 test6 test7
+PYANG := $(or $(PYANG), pyang)
```

Comment:
This line (there's a similar line in `test_yang/Makefile`) is just to ease standalone testing. It doesn't affect the automated test environment, which predefines the `PYANG` variable.